### ISSUE TITLE
After successful supervised MultilevelSwitchCCSet (255) query the actual value quicker 

### ISSUE THREAD

**Issue by AlCalzone** [AlCalzone/node-zwave-js#5124]
```
2022-09-26T14:38:44.203Z DRIVER « [Node 004] [REQ] [ApplicationCommand]
└─[MultiChannelCCCommandEncapsulation]
│ source: 1
│ destination: 0
└─[SupervisionCCReport]
session id: 19
more updates follow: false
status: Success
duration: 0s
```

We don't need to wait 5 seconds after such a report.

Starting point:
`packages/cc/src/cc/MultilevelSwitchCC.ts` --> `SET_VALUE_HOOKS`

**Comment by PhilipTrauner** [comment]
would like to tackle this 🙋
diving into the code right now, will report back once i get a feel for things

**Comment by PhilipTrauner** [comment]
i think i have a general feeling of how this works, and where the five seconds come from.

`CommandClass`es can hook into the process of [setting values on a node](https://github.com/zwave-js/zwave-js/blob/7f04bb602bded4ca0d8edb4ae60e319f438e99a8/packages/zwave-js/src/lib/node/Node.ts#L458) by implementing  [`SET_VALUE_HOOKS` ](https://github.com/zwave-js/zwave-js/blob/7f04bb602bded4ca0d8edb4ae60e319f438e99a8/packages/cc/src/lib/API.ts#L321) (which is exposed "up the stack" through [`setValueHooks()`](https://github.com/zwave-js/zwave-js/blob/7f04bb602bded4ca0d8edb4ae60e319f438e99a8/packages/cc/src/lib/API.ts#L337)). 
the method takes in multiple parameters, including `options`, which is built in [`setValue`](https://github.com/zwave-js/zwave-js/blob/7f04bb602bded4ca0d8edb4ae60e319f438e99a8/packages/zwave-js/src/lib/node/Node.ts#L517). 
the optional `transitionDuration` property of `options`, [which dictates how far in the future a poll should be scheduled](https://github.com/zwave-js/zwave-js/blob/7f04bb602bded4ca0d8edb4ae60e319f438e99a8/packages/cc/src/cc/MultilevelSwitchCC.ts#L388), is obtained through the [`NetworkCache`](https://github.com/zwave-js/zwave-js/blob/7f04bb602bded4ca0d8edb4ae60e319f438e99a8/packages/zwave-js/src/lib/node/Node.ts#L422) (*not* ephemeral, stored as node-specific `defaultTransitionDuration` key) associated with the `Node`'s `Driver`. 

as far as i can tell this value will only ever be set manually, and in the case of it being unset, [`schedulePoll`](https://github.com/zwave-js/zwave-js/blob/7f04bb602bded4ca0d8edb4ae60e319f438e99a8/packages/cc/src/lib/API.ts#L362) obtains the hosts [`getRefreshValueTimeouts`](https://github.com/zwave-js/zwave-js/blob/7f04bb602bded4ca0d8edb4ae60e319f438e99a8/packages/zwave-js/src/lib/driver/Driver.ts#L1360), which provides a `refreshValue`, and a `refreshValueAfterTransition`. 
which one of those is used depends on the `transition` property of the `options` parameter, which falls back to `"slow"` when not provided, as is the case with `MultilevelSwitchCC`. 
in that instance `refreshValue` is used, [which happens to be 5 seconds](https://github.com/zwave-js/zwave-js/blob/7f04bb602bded4ca0d8edb4ae60e319f438e99a8/packages/zwave-js/src/lib/driver/Driver.ts#L320).



long story short, i see two places where the five second interval could be adjusted:

* [as part of the `Driver`](https://github.com/zwave-js/zwave-js/blob/7f04bb602bded4ca0d8edb4ae60e319f438e99a8/packages/zwave-js/src/lib/driver/Driver.ts#L320)

* [as the fallback value in `schedulePoll`](https://github.com/zwave-js/zwave-js/blob/7f04bb602bded4ca0d8edb4ae60e319f438e99a8/packages/cc/src/lib/API.ts#L369)

  

i can't claim to be familiar enough with the Z-Wave to determine if replacing one hardcoded value with another will have any adverse effects, or what that new value should be.



what do you think @AlCalzone? would it make sense to start off with a short default interval which is then further shortened / lengthened until changes can be observed?

**Comment by AlCalzone** [comment]
I think you have a good idea how it fits together. The main idea of this issue is that once we receive the above report (which you can observe in the `ZWaveNode.setValue` call), we shouldn't have to wait at all - the Supervision Report indicates that the device is done, but since we set the value to 255 (on), we don't know the actual value (1-99). This should probably be done by skipping the scheduling machinery and polling the value directly, but I'll let you come up with a proposal for the specifics.

**Comment by PhilipTrauner** [comment]
at the risk of having overlooked something, wouldn't calling `CCAPI.pollValue` instead of `CCAPI.schedulePoll` in [`verifyChanges`](https://github.com/zwave-js/zwave-js/blob/570fa667e41853fc594a7ded53972ae354649b99/packages/cc/src/cc/MultilevelSwitchCC.ts#L471) be sufficient then?

[`schedulePoll`](https://github.com/zwave-js/zwave-js/blob/570fa667e41853fc594a7ded53972ae354649b99/packages/zwave-js/src/lib/node/mixins/60_ScheduledPoll.ts#L113) [already calls `pollValue` internally](https://github.com/zwave-js/zwave-js/blob/570fa667e41853fc594a7ded53972ae354649b99/packages/zwave-js/src/lib/node/mixins/60_ScheduledPoll.ts#L149), therefor i assume there would be no difference from the perspective of the request handling machinery (`Driver.handleRequest` → `Node.handleCommand` → `CCHandlers.handleMultilevelSwitchCommand` → `ValueDB.setValue`), right?

**Comment by AlCalzone** [comment]
I think that would be okay - for the specific case of successful supervised commands.
